### ISSUE TITLE
Seul le modèle Arôme est disponible

### ISSUE THREAD

**Issue by Obuis** [mmomtchev/velivole#4]
Il n’y a plus que le modèle Arôme. Les modèle ICON, ARPÈGE et GFS ont disparu

**Comment by mmomtchev** [comment]
Tout à fait, je ne peux plus me permettre de payer la location des serveurs qui permettait d'avoir tous les modèles, le message était très explicite depuis deux mois.

Depuis des années quelques sociétés font du chantage - un chantage qui touche bien le monde du parapente - pour couvrir une affaire extrêmement embarrassante devant les tribunaux - à cause du caractère affectif, voire sexuel, de sa motivation.

Ceci est la raison pour laquelle velivole.fr n'a toujours pas de sponsor.

Si on arrive à trouver une/des association(s)/clubs/fabricant de matériel qui soient prêt à financer le site à la hauteur d'au moins 50€/mois, il pourra revenir dans son format d'avant.

Son hébergement actuel coûte 8€/mois et c'est le maximum absolu que je peux me permettre d'investir dedans.

Je suis actuellement au RSA et un nombre de cabinets de recrutement se sont joint au chantage et me demandent de ne plus parler de cette affaire pour que je puisse retravailler.

**Comment by mmomtchev** [comment]
On me propose des financements du type ***Héritage d'un prince nigérian*** dont le provenance est bien évidemment la société derrière cette affaire. Je ne peux pas les accepter - même si le prince nigérian était un militant pour les droit de l'homme - ce qui semble le type d'argument principal qu'on avance.
